ci/playbooks/build_runner_image.yaml

```diff
@@ -36,4 +36,6 @@
         content: |
           make_edpm_deploy_env:
             DATAPLANE_RUNNER_IMG: "{{ ansibleee_runner_img }}"
+          make_edpm_compute_baremetal:
```

Comment:
Is this correct? should not this be `make_edpm_compute_baremetal_env`.  Btw, where is the edpm-ansible.yml created here passed in ci-framework?

Reply:
OK I see it in https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/zuul.d/jobs.yaml#L40

Reply:
Thank you @rabi for fixing it.